python/ray/_raylet.pyx

```diff
@@ -3890,6 +3890,7 @@ cdef class CoreWorker:
                      concurrency_groups_dict,
                      int32_t max_pending_calls,
                      scheduling_strategy,
+                     c_bool report_task_events,
```

Comment:
Has this pull request not implemented specifying "report_task_events" when creating an actor? `report_task_events` is not used.

Reply:
yes, this is still WIP - finalizing on the APIs and design.